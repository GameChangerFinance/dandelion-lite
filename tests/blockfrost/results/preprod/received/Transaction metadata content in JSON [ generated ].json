[
  {
    "json_metadata": {
      "6cce2ea44248956802eb41677c753014baa9f8a53c95c191d58fea7a": {
        "Test 7421": {
          "description": "",
          "files": [
            {
              "mediaType": "image/png",
              "name": "Test 7421",
              "src": "ipfs://QmSgwxB157u7GTCYA6hh6U5bUkWJAiUurA4VccBxaaprgW"
            }
          ],
          "image": "ipfs://QmSgwxB157u7GTCYA6hh6U5bUkWJAiUurA4VccBxaaprgW",
          "mediaType": "image/png",
          "name": "Test 7421"
        }
      },
      "version": "1.0"
    },
    "tx_hash": "388557c89195fb6bcf7fdfaa2ca2cb6f21099805cf31415b152ea8b350c3e12d"
  },
  {
    "json_metadata": {
      "6cce2ea44248956802eb41677c753014baa9f8a53c95c191d58fea7a": {
        "Test 6338": {
          "description": "",
          "files": [
            {
              "mediaType": "image/png",
              "name": "Test 6338",
              "src": "ipfs://QmSgwxB157u7GTCYA6hh6U5bUkWJAiUurA4VccBxaaprgW"
            }
          ],
          "image": "ipfs://QmSgwxB157u7GTCYA6hh6U5bUkWJAiUurA4VccBxaaprgW",
          "mediaType": "image/png",
          "name": "Test 6338"
        }
      },
      "version": "1.0"
    },
    "tx_hash": "4b02516b270d876a81d57bceeccf1a1c657fd8c124e87c0b657e9283a885f171"
  },
  {
    "json_metadata": {
      "6cce2ea44248956802eb41677c753014baa9f8a53c95c191d58fea7a": {
        "Test 1012": {
          "description": "",
          "files": [
            {
              "mediaType": "image/png",
              "name": "Test 1012",
              "src": "ipfs://QmSgwxB157u7GTCYA6hh6U5bUkWJAiUurA4VccBxaaprgW"
            }
          ],
          "image": "ipfs://QmSgwxB157u7GTCYA6hh6U5bUkWJAiUurA4VccBxaaprgW",
          "mediaType": "image/png",
          "name": "Test 1012"
        }
      },
      "version": "1.0"
    },
    "tx_hash": "12bba31d846f54739a8e5bb722b7aeceed5ecdcb0948cb74c1dcce63352b0342"
  }
]
