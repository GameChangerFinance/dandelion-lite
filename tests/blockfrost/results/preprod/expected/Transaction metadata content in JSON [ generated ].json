[
  {
    "json_metadata": {
      "6cce2ea44248956802eb41677c753014baa9f8a53c95c191d58fea7a": {
        "Test 1283": {
          "description": "",
          "files": [
            {
              "mediaType": "image/png",
              "name": "Test 1283",
              "src": "ipfs://QmSgwxB157u7GTCYA6hh6U5bUkWJAiUurA4VccBxaaprgW"
            }
          ],
          "image": "ipfs://QmSgwxB157u7GTCYA6hh6U5bUkWJAiUurA4VccBxaaprgW",
          "mediaType": "image/png",
          "name": "Test 1283"
        }
      },
      "version": "1.0"
    },
    "tx_hash": "965b9d64386587f5e89e7261d986433fcf042707d9553312d5f230442216090f"
  },
  {
    "json_metadata": {
      "6cce2ea44248956802eb41677c753014baa9f8a53c95c191d58fea7a": {
        "Test 7707": {
          "description": "",
          "files": [
            {
              "mediaType": "image/png",
              "name": "Test 7707",
              "src": "ipfs://QmSgwxB157u7GTCYA6hh6U5bUkWJAiUurA4VccBxaaprgW"
            }
          ],
          "image": "ipfs://QmSgwxB157u7GTCYA6hh6U5bUkWJAiUurA4VccBxaaprgW",
          "mediaType": "image/png",
          "name": "Test 7707"
        }
      },
      "version": "1.0"
    },
    "tx_hash": "a01a4140a989bff620c668ad5c7311481b3b3088b2d77a80d4b51b47c20e6e68"
  },
  {
    "json_metadata": {
      "de4d825e151f7f9ed89233c04e6e7e29f4372e125a3fff57a674d0d3": {
        "56462d36": {
          "description": [
            "Social\nSocial\nSocial\nSocial\nSocial\nSocial\nSocial\nSocial\nSocial\nS",
            "ocial\nSocial\nSocial\nSocial\nSocial\nSocial\nSocial\nSocial\nSocial\nSo",
            "cial\nSoc"
          ],
          "files": [
            {
              "mediaType": "image/png",
              "name": "VF-6",
              "src": "ipfs://QmUa3Fnw5oiE9xcRoGbhY7FvdgWeqxcjwxkBs2R1JiV93R"
            }
          ],
          "image": "ipfs://QmUa3Fnw5oiE9xcRoGbhY7FvdgWeqxcjwxkBs2R1JiV93R",
          "mediaType": "image/png",
          "name": "VF-6"
        }
      }
    },
    "tx_hash": "446404d43328ef6e6427a99e5b1d0c1b9f530242e221060d86e8cda730aa7058"
  }
]
